src/Features/Core/Portable/MakeMethodAsynchronous/AbstractMakeMethodAsynchronousCodeFixProvider.cs

```diff
@@ -100,8 +101,10 @@ internal abstract class AbstractMakeMethodAsynchronousCodeFixProvider : CodeFixP
             var semanticModel = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
             var methodSymbolOpt = semanticModel.GetDeclaredSymbol(node) as IMethodSymbol;
 
-            if (methodSymbolOpt?.MethodKind == MethodKind.Ordinary &&
-                !methodSymbolOpt.Name.EndsWith(AsyncSuffix))
+            bool isOrdinaryOrLocalFunction =
+                methodSymbolOpt?.MethodKind == MethodKind.Ordinary
+                || methodSymbolOpt?.MethodKind == MethodKind.LocalFunction;
```

Comment:
our general pattern is to keep && and || with the left side of the binary and to align the left and right when they wrap.